CONTRIBUTING.md

```diff
@@ -0,0 +1,186 @@
+```bash
+cd ~/ansible_collections/community/mysql
+```
+
+8. Be sure you are in the main branch:
+```bash
+git status
+```
+
+9. Show remotes. There should be the `origin` repository only:
+```bash
+git remote -v
+```
+
+10. Add the `upstream` repository:
+```bash
+git remote add upstream https://github.com/ansible-collections/community.mysql.git
+```
+
+11. Update your local `main` branch:
+```bash
+git fetch upstream
+git rebase upstream/main
+```
+
+12. Create a branch for your changes:
+```bash
+git branch -b name_of_my_branch
+```
+
+13. It can be a good approach is to start with writing integration tests if applicable.
+
+Note: If there are any difficulties with writing the tests or you are not sure if the case can be covered, feel free to skip this step.
+If needed, other contributors can help you with it later.
+
+All integration tests are stored in `tests/integration/targets` subdirectories.
+You are interested in a subdirectory containing a name of module you are going to change.
+For example, if you are fixing the `mysql_user` module, its tests are in `tests/integration/targets/test_mysql_user/tasks`
+
+There is the `main.yml` file which includes other test files.
+Look for a suitable one to integrate your tests or create and include a dedicated one.
+You can use one of the existing as a draft.
+
+When fixing a bug, write a task which reproduces the bug from the issue.
+
+Put the reported case in the tests, then run integration tests with the following command:
+```bash
+ansible-test integration test_mysql_user --docker -vvv > ~/test.log
+```
+If the tests do not want to run, first, check you complete step 3 of this guide.
+
+If the tests ran successfully, there are usually two possible outcomes:
+a) If the bug has not appeared and the tests have passed successfully, ask the reporter to provide more details. The bug can be not a bug actually or can relate to a particular software version used or specifics of local environment configuration.
+
+b) The bug has appeared and the tests has failed as expected showing the reported symptoms.
+
+14. Fix the bug.
+
+15. Run `flake8` against a changed file. If it is `plugins/modules/mysql_user.py`:
+```bash
+flake8 plugins/modules/mysql_user.py
+```
+It is worth installing and running `flake8` against the changed file(s) first.
+It shows unused imports, which is not shown by sanity tests (see the next step), as well as other common issues.
+Optionally, you can use the `--max-line-length=160` command-line argument.
+
+16. Run sanity tests:
+```bash
+ansible-test sanity plugins/modules/mysql_user.py --docker
+```
+If they failed, look at the output carefully - it is usually very informative and helps to identify a problem line quickly.
+Sanity failings usually relate to wrong code and documentation formatting.
+
+17. Run integration tests:
+```bash
+ansible-test integration test_mysql_user --docker -vvv
+```
+
+There are two possible outcomes:
+a) They have failed. Look into the `test.log`.
```

Comment:
```suggestion
a) They have failed. Look at the output of your command.
```

Reply:
good spot! maybe `output of the command`?